linkerd/main/src/main/scala/io/buoyant/linkerd/Main.scala

```diff
@@ -33,6 +34,9 @@ object Main extends App {
     val build = Build.load("/io/buoyant/linkerd/build.properties")
     log.info("linkerd %s (rev=%s) built at %s", build.version, build.revision, build.name)
 
+    // #2166 -- Linkerd logs incorrect message about Content-Length for HTTP 204 responses
```

Comment:
A link in this comment to the underlying Finagle and/or Netty issues that cause this would be good.

Reply:
@adleong Hey I updated the issue to link back to finagle